integrations/helm/release/utils_test.go

```diff
@@ -0,0 +1,168 @@
+	"image": "bitnami/kibana:master-1",
+}
+
+var mappifyExpectedNested2 = map[string]interface{}{
+	"image": map[string]interface{}{
+		"repository": "bitnami/kibana:master-1",
+	},
+}
+
+var mappifyExpectedNested3 = map[string]interface{}{
+	"image": map[string]interface{}{
+		"repository": map[string]interface{}{
+			"name": "bitnami/kibana:master-1",
+		},
+	},
+}
+
+// fixtures for collectValues
+var collectSimpleSimpleParams = []ifv1.HelmChartParam{
+	{Name: "image", Value: "bitnami/kibana"},
+	{Name: "tag", Value: "alpha"},
+	{Name: "memory", Value: "1Gi"},
+}
+var collectExpectedSimpleSimple = map[string]interface{}{
+	"image":  "bitnami/kibana",
+	"tag":    "alpha",
+	"memory": "1Gi",
+}
+
+var collectSimpleListParams = []ifv1.HelmChartParam{
+	{Name: "image", Value: "bitnami/kibana"},
+	{Name: "tag", Value: "alpha"},
+	{Name: "tolerances", Value: `[{"key":"aaa","value":"val"},{"key":"bbb","value":"val2"}]`},
+}
+var collectExpectedSimpleList = map[string]interface{}{
+	"image":      "bitnami/kibana",
+	"tag":        "alpha",
+	"tolerances": []interface{}{map[string]interface{}{"key": "aaa", "value": "val"}, map[string]interface{}{"key": "bbb", "value": "val2"}}}
+
+var collectNestedSimpleParams = []ifv1.HelmChartParam{
+	{Name: "image.repository.name", Value: "bitnami/kibana"},
+	{Name: "resources.memory", Value: "1Gi"},
+}
+var collectExpectedNestedSimple = map[string]interface{}{
+	"image": map[string]interface{}{
+		"repository": map[string]interface{}{
+			"name": "bitnami/kibana",
+		},
+	},
+	"resources": map[string]interface{}{
+		"memory": "1Gi",
+	},
+}
+
+var collectNestedListParams = []ifv1.HelmChartParam{
+	{Name: "image.repository.name", Value: `[{"key":"aaa","value":"val1"},{"key":"bbb","value":"val2"}]`},
+	{Name: "resources.memory", Value: `[{"key":"ccc","value":"val3"},{"key":"ddd","value":"val4"}]`},
+}
+var collectExpectedNestedList = map[string]interface{}{
+	"image":     map[string]interface{}{"repository": map[string]interface{}{"name": []interface{}{map[string]interface{}{"key": "aaa", "value": "val1"}, map[string]interface{}{"key": "bbb", "value": "val2"}}}},
+	"resources": map[string]interface{}{"memory": []interface{}{map[string]interface{}{"key": "ccc", "value": "val3"}, map[string]interface{}{"key": "ddd", "value": "val4"}}}}
+
+var collectNestedMixtureParams = []ifv1.HelmChartParam{
+	{Name: "image.repository.name", Value: "bitnami/kibana"},
+	{Name: "resources.memory", Value: `[{"key":"aaa","value":"val"},{"key":"bbb","value":"val2"}]`},
+}
+var collectExpectedNestedMixture = map[string]interface{}{
+	"image": map[string]interface{}{
+		"repository": map[string]interface{}{
+			"name": "bitnami/kibana",
+		},
+	},
+	"resources": map[string]interface{}{"memory": []interface{}{map[string]interface{}{"key": "aaa", "value": "val"}, map[string]interface{}{"key": "bbb", "value": "val2"}}}}
+
+// fixtures for unwrap tests
+func TestMappifyValueOverride(t *testing.T) {
+	for _, c := range []mappifyTestCase{
```

Comment:
would it be easier to read if the fixtures were in the test function? they don't seem to be shared and currently have to scroll up and down

Reply:
I moved the fixtures towards the tests, so hopefully more readable.  If it still bothers you greatly I shall insert the fixtures inside (this was done from start for the unwrap tests as the test input is small). Though I would prefer not to have to do it :)